src/elements/tabs/tab-label/tab-label.visual.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { html } from 'lit';
+
+import {
+  describeEach,
+  describeViewports,
+  visualDiffDefault,
+  visualDiffStandardStates,
+} from '../../core/testing/private.js';
+
+import '../tab-group.js';
+import '../tab-label.js';
+import '../tab.js';
+
+const cases = {
+  amount: ['', 16],
+  iconName: ['', 'app-icon-small'],
+};
+
+describe(`sbb-tab-label`, () => {
+  describeViewports({ viewports: ['zero', 'medium'] }, () => {
+    for (const state of visualDiffStandardStates) {
+      it(
+        state.name,
```

Comment:
the active state in Firefox seems not to work, but I think we can accept this :)